src/main.ts

```diff
@@ -617,21 +617,21 @@ class GeneralSettingsTab extends PluginSettingTab {
                         this.display();
                     })
                 );
-        else taskEmojiShorthand.setDesc('Only available when "Automatic Task Completion Tracking" is enabled.');
+        else taskEmojiShorthand.setDesc('Only available when "automatic task completion tracking" is enabled.');
 
         let taskFieldName = new Setting(this.containerEl)
-            .setName("Completion Field Name")
+            .setName("Completion field name")
             .setDisabled(!taskCompletionInlineSubsettingsEnabled);
         if (taskCompletionInlineSubsettingsEnabled)
             taskFieldName
                 .setDesc(
                     createFragment(el => {
                         el.appendText(
-                            "Text used as inline field key for task completion date when toggling a task's checkbox in a dataview view."
+                            "Text used as inline field key for task completion date when toggling a task's checkbox in a Dataview view."
                         );
                         el.createEl("br");
                         el.appendText(
-                            'Only available when "Automatic Task Completion Tracking" is enabled and "Use Emoji Shorthand for Completion" is disabled.'
+                            'Only available when "automatic task completion tracking" is enabled and "use emoji shorthand for completion" is disabled.'
```

Comment:
Shouldn't automatic be Automatic as were referring to the setting itself? This is the height of pedantry lol but might as well.